docs/versions/v0.0.2/vc-http-api.yml

```diff
@@ -468,11 +468,11 @@ components:
           type: object
           description: The holder - will be ignored if no proof is present since there is no proof of authority over the credentials
           nullable: true
-        "verifiableCredential":
+        "verifiableCredentials":
```

Comment:
This would be a violation of the VC Data Model, wouldn't it? https://w3c.github.io/vc-data-model/#presentations-0

Reply:
yep, linked data "convention" is to use "singular" for arrays, and camelCase for field names... 2 boats that have sailed long ago, in a direction of less readability :)

none the less, the convention (and spec conformance) MUST be preserved.


> verifiableCredential
> If present, the value of the verifiableCredential property MUST be constructed from one or more verifiable credentials, or of data derived from verifiable credentials in a cryptographically verifiable format.


technically `verifiableCredential` is not required and `verifiableCredentials` is not forbidden, but we can all see how terrible pulling on that thread would be.

Reply:
```suggestion
        "verifiableCredential":
```